### PR TITLE
Julcsi

### PR DESCRIPTION
**Pull request by julcsi333**
Valamelyikőtök megnézné esetleg hogy működik? Nálam elvileg működött

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
...
 
-## **Container**
\ No newline at end of file
+## **Container**
+
+### Requirements
+The project can be run through a Docker container. Docker has to be installed on computer, for this to work.
+
+### Start docker
+- After docker is install, simply run **docker_start.ps1** script in a terminal:
```
Comment (annalaczko) [inline README.md:41]: *installed

File: README.md [hunk 2]
```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
 
 With this you should be able to run the *Cardiac MRI segmentation-checkpoint.ipynb*.
 
-## **Container**
\ No newline at end of file
+## **Container**
+
+### Requirements
+The project can be run through a Docker container. Docker has to be installed on computer, for this to work.
```
Comment (annalaczko) [inline README.md:38]: The project can be run with a Docker container. Docker has to be installed on this computer, for this to work.

File: README.md [hunk 3]
```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
...
+
+### Start docker
+- After docker is install, simply run **docker_start.ps1** script in a terminal:
+`.\docker_start.ps1`
+- Click the url provided by the docker in the terminal to reach the jupyter server
```
Comment (annalaczko) [inline README.md:43]: - Click the url provided by Docker in the terminal to reach the jupyter server

File: README.md [hunk 4]
```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
...
+e.g.: `http://127.0.0.1:8888/tree?token=56646...`
+- Click `Cardiac_MRI_segmentation-checkpoint.ipynb`
+- Saving in browser (jupyter) also saves the file filesystem
```
Comment (annalaczko) [inline README.md:46]: Saving in browser (jupyter) also saves the file in the filesystem

File: README.md [hunk 5]
```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
...
+- Changing the files in filesystem also change files in docker (refresh in browser updates files in jupyter)
```
Comment (annalaczko) [inline README.md:47]: Changing the files in the filesystem also changes files in Docker (refreshing the browser updates the files in jupyter)

**Review by annalaczko — CHANGES_REQUESTED** [review]
Corrected some grammar anomalies in README. Had no time to check docker.

File: README.md (hunk 2 above)
Comment (julcsi333) [reply README.md:38]: changed to "run in a Docker container"

File: README.md (hunk 3 above)
Comment (julcsi333) [reply README.md:43]: changed to "by the docker container" because it sounds better

**Pull request merged**